org.lflang/src/org/lflang/generator/TypeScriptGenerator.xtend

```diff
@@ -57,6 +58,7 @@ import org.lflang.lf.VarRef
 import org.lflang.lf.Variable
 
 import static extension org.lflang.ASTUtils.*
+import org.lflang.federated.SupportedSerializers
```

Comment:
TypeScript generator is a Kotlin class in master...

Reply:
I'm not sure how this class ended up being here. Should I just remove it in this branch?

Reply:
I believe `TypeScriptGenerator.xtend` has never been removed it look like. It is still on master...

Reply:
Ah, you're right.